### PR TITLE
Google::Protobuf::Struct can access a missing key (#3846)

### PR DESCRIPTION
**Pull request by FX-HAO**
#3846

### CODE REVIEW

**Comment by FX-HAO** [comment]
I signed it!

**Comment by FX-HAO** [comment]
I signed it!

File: ruby/tests/well_known_types_test.rb [hunk 1]
```diff
@@ -82,6 +82,9 @@ def test_struct
     # to_h returns a fully-flattened Ruby structure (Hash and Array).
     assert_equal(should_equal, struct.to_h)
 
+    # Test that we can safely access a missing key
+    assert_equal(nil, struct['nonexistence'])
```
Comment (haberman) [inline ruby/tests/well_known_types_test.rb]: Please match quote style of the rest of the file.

**Comment by FX-HAO** [comment]
@haberman Updated it.

**Comment by FX-HAO** [comment]
@haberman Hi haberman! It's a long time since I corrected the PR according to your feedback. Can you merge it?

**Comment by anandolee** [comment]
Ping @haberman

**Comment by haberman** [comment]
Very sorry for the delay.

**Pull request merged**